mongoSeed/init/create-db.js

```diff
@@ -0,0 +1,9 @@
+db = new Mongo().getDB('purchase-db');
+
+db.createUser({
+  user: "purchase-admin",
```

Comment:
lets make sure that in our implementation that we're not using this username/pwd in openshift please :)